src/index.js

```diff
@@ -0,0 +1,338 @@
+import {
+  setListItemsStorage,
+  setListStorage,
+  getLocalStorage,
+  setLocalStorage,
+  listItemsStorage,
+  updateLocalStorage,
+  listStorage,
+  listLength,
+} from './storage';
+
+
+const projectInput = document.getElementById('project');
+const createProject = document.getElementById('create-project');
+const ul = document.querySelector('.list-group');
+const rightBtn = document.getElementById('slide-right');
+const leftBtn = document.getElementById('slide-left');
+
+
+setListItemsStorage();
+setListStorage();
+
+let currentProject = -1;
+
+function Project(title) {
+  this.title = title;
+  let idx;
+  if (listLength(listStorage) === 0) {
+    idx = 0;
+  } else {
+    idx = JSON.parse(localStorage.getItem(listStorage))[listLength(listStorage) - 1].id + 1;
+  }
+  this.id = idx;
+}
+
+function Item(parentId, title, description, date, priority) {
+  this.parentId = parentId;
+  this.title = title;
+  this.date = date;
+  this.description = description;
+  this.priority = priority;
+  this.status = false;
+  let idx;
+  if (listLength(listItemsStorage) === 0) {
+    idx = 0;
+  } else {
+    idx = JSON.parse(localStorage.getItem(listItemsStorage))[listLength(listItemsStorage) - 1].id + 1; /* eslint-disable-line */
+  }
+  this.id = idx;
+}
+
+
+createProject.addEventListener('click', () => {
+  if (projectInput.value) {
+    const newProject = new Project(projectInput.value);
+    setLocalStorage(getLocalStorage(listStorage), listStorage, newProject);
+    projectInput.value = '';
+    window.location.reload();
+  }
+});
+
+function render() {
```

Comment:
- [ ] You have a file for the DOM manipulation that's empty, why don't you put this and other DOM related functions there?